src/wp-includes/class-wp-query.php

```diff
@@ -3292,8 +3292,21 @@ public function get_posts() {
 			 *
 			 * @param bool     $split_the_query Whether or not to split the query.
 			 * @param WP_Query $query           The WP_Query instance.
+			 * @param string   $old_request     The complete SQL query before filtering.
```

Comment:
This looks to be available at `$query->request` at this point.

Reply:
Old query is the value BEFORE the filter are run, meaning you can't get this.

Reply:
> Old query is the value BEFORE the filter are run, meaning you can't get this.

OK, thanks for the clarification.